tools/openqa-run

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -euo pipefail
+
+usage() {
+    cat << EOF
+Usage: openqa-run [OPTIONS...] OPENQA_SERVICE
+Call an openQA service OPENQA_SERVICE with appropriate user account and the
+base-dir set to the local working copy.
+
+Example:
+    openqa-run openqa-webui-daemon
+
+Options:
+ -h, --help         display this help
+EOF
+    exit "$1"
+}
+
+OPENQA_BASEDIR="${OPENQA_BASEDIR:-"$(dirname "$0")/../t/data"}"
```

Comment:
Not sure whether that's the best idea. I get the idea of reusing test fixtures. However, it is also a bit problematic as this might lead to accidental cluttering and changes in that directory. This might be annoying because those changes might show up in Git (and we cannot just ignore them because files in that directory are supposed to be under version control) and might also lead to failures when running unit tests (which might not be obvious to debug).

Reply:
We could create a temporary directory and use that.